blackhole.go

```diff
@@ -46,6 +48,11 @@ func (client *BlackholeClient) PublishAuditLog(auditLogBuilder *AuditLogBuilder)
 	return nil
 }
 
+func (client *BlackholeClient) PublishMessage(topic, messageKey string, message interface{}, errorCallback PublishErrorCallbackFunc) error {
```

Comment:
I suggest that you build a specific user-streaming method, provided you have specific message payload requirements.

Reply:
@angelo-chan 
do you mean a function like PublishUserAccountHistory(userAccountHistory UserAccountHistory)
and the UserAccountHistory would look like:

type UserAccountHistory struct {
    AuditLog
}